modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -584,6 +592,8 @@ feature {NONE} -- Queries
 
 	sql_delete_node_revisions: STRING = "DELETE FROM node_revisions WHERE nid=:nid;"
 
+	sql_delete_node_aliases: STRING = "DELETE FROM path_aliases WHERE source='$nodeid';"
```

Comment:
because the source has the following pattern "node/id" but I could compute it and set as :nid